lib/eval-snippets.ts

```diff
@@ -49,6 +49,8 @@ export const snippets = {
   },
   EVAL_ONETRUST_1: () => window.OnetrustActiveGroups.split(',').filter(s => s.length > 0).length <= 1,
   EVAL_TRUSTARC_TOP: () => window && window.truste && window.truste.eu.bindMap.prefCookie === '0',
+  EVAL_TRUSTARC_FRAME: () => window && window.QueryString && window.QueryString.preferences === '0',
```

Comment:
I know this is not followed everywhere, but it'd be nice to use a more meaningful name for snippets. Something like "EVAL_TRUSTART_FRAME_TEST"?